tensorflow/python/ops/special_math_ops.py

```diff
@@ -82,7 +82,7 @@ def lbeta(x, name='lbeta'):
     return result
 
 
-def einsum(equation, *inputs):
+def einsum(equation, *inputs, **kwargs):
```

Comment:
It appears to be unavoidable, see https://stackoverflow.com/questions/20961100/passing-default-parameters-with-varargs-in-function-in-python